component/lambda/sql/billing.sql

```diff
@@ -3,15 +3,6 @@ from data catalog
 database 'si-prod-data'
 iam_role 'arn:aws:iam::944151301248:role/si-prod-redshift,arn:aws:iam::798856999590:role/si-prod-glue-data-bucket';
```

Comment:
chore: might be worth pushing this to an env var, it's got our account IDs in it

Reply:
Oh geez! Thanks!